app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import { CONTACTS_VALIDATION, ROLES_VALUES } from "../constants";
+
+const ContactForm = ({ onClose, contact, isEdit }) => {
```

Comment:
Go through the Slack thread.

Reply:
Should I update the file name to match the component name as we are also utilising the Formik form component here, or should I use an alias for Formik form?

Reply:
<img width="962" alt="image" src="https://user-images.githubusercontent.com/104645943/192725144-909b93dd-d020-4603-9c6e-7493a2ab6e9c.png">

In table component, we have used neetoui Table as NeetoUITable. So, should I approach in the same way for Form component?

Reply:
@Ishaan-Singhal Please go through https://github.com/bigbinary/bigbinary-website/issues/2462 and lemme know if that answers your queries. If not, lemme know.